packages/plugins/test/mocha/foundry/decode.spec.ts

```diff
@@ -34,7 +34,7 @@ describe('Decoded Actions', () => {
     },
   }
 
-  describe('makeCreate2DecodedAction', () => {
+  describe('makeContractDecodedAction', () => {
```

Comment:
This function applies to both `create` and `create2` deployments, which is why I changed the function's name